src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenInteractiveParameters.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Identity.Client.ApiConfig.Parameters
 
     internal class AcquireTokenInteractiveParameters : IAcquireTokenParameters
     {
-        public Prompt Prompt { get; set; } = Prompt.SelectAccount;
+        public Prompt Prompt { get; set; } = Prompt.NotSpecified;
```

Comment:
this is interesting. is this a breaking change?

Reply:
Nice catch 👍 
This is not a breaking change, as "NotSpecified" is translated to "SelectAccount" when broker is not in use. I've noticed that we don't really offer the users the possibility of not sending a promp option (except via NoPrompt which is documented as for B2C use only), so maybe we will want to take this a breaking change. I logged https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2086 to track this.